radicle-cob/Cargo.toml

```diff
@@ -17,6 +17,7 @@ git-commit = { version = "0.2" }
 git-ref-format = { version = "0.1" }
 git-trailers = { version = "0.1" }
 log = { version = "0.4.17" }
+nonempty = { version = "0.8.1", git = "https://github.com/cloudhead/nonempty.git", rev = "cdf611bab2bc37ff9c443067638324fb32a21f2e", features = ["serialize"] }
```

Comment:
Published it :)

Reply:
Is there a reason for the revision to be specified?  It may impact how cargo uses semantic versioning to apply fixes.  When and if a fix comes out, say version 0.8.2, cargo could build the project with the latest patched version.

Reply:
It will build with `0.8.2` but this says the minimum is `0.8.1` which is true since we're using methods that only exist there.

Reply:
@FintanH my concern is about the 'rev = "cdf611bab2bc37ff9c443067638324fb32a21f2e"' affecting this.  Would it not?

Reply:
Right, sorry that was temporary as I had opened a PR on `nonempty`. @cloudhead fixed this up for me before merging.